plugins/woocommerce-blocks/tests/e2e/playwright-utils/test.ts

```diff
@@ -154,6 +160,15 @@ const test = base.extend<
 	storeApiUtils: async ( { requestUtils }, use ) => {
 		await use( new StoreApiUtils( requestUtils ) );
 	},
+	shippingUtils: async ( { page }, use ) => {
+		await use( new ShippingUtils( page ) );
+	},
+	localPickupUtils: async ( { page }, use ) => {
+		await use( new LocalPickupUtils( page ) );
+	},
+	miniCartUtils: async ( { page }, use ) => {
+		await use( new MiniCartUtils( page ) );
```

Comment:
Since `miniCartUtils` uses `frontEndUtils` can we pass `this.frontendUtils` to the constructor to prevent passing it in the `openMiniCart` method?

Reply:
See https://github.com/woocommerce/woocommerce/pull/44108#discussion_r1469451202